molgenis-data-security/src/main/java/org/molgenis/data/security/auth/GroupService.java

```diff
@@ -86,6 +102,126 @@ public void persist(GroupValue groupValue)
 		dataService.add(ROLE, roles.stream());
 	}
 
+	/**
+	 * Grants default permissions on the root package and group to the roles of the group
+	 *
+	 * @param groupValue details of the group for which the permissions will be granted
+	 */
+	public void grantPermissions(GroupValue groupValue)
```

Comment:
'grantDefaultPermissions' seems like a more descriptive name. I would suggest moving the create actions from the controller to the service , this can then be part of the service method and be kept private ( inner workings of creating a group)

Reply:
Method used to sit at the bottom of the class.
This PR moves it to the top, and makes it grant two permission sets, on both group and package, instead of just on the package.

Reply:
Would actually be nice if we can get rid of it here altogether and let another class worry about the permissions on the group? Can also get rid of the PermissionService as a constructor parameter here then.